composer.json

```diff
@@ -38,8 +36,12 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.15",
         "localheinz/composer-normalize": "~0.8.0",
+        "php-http/httplug": "^1.0 || ^2.0",
```

Comment:
As we will release this feature in a minor version (5.1), we can't remove `php-http/*` dependencies until next major